judgments/utils.py

```diff
@@ -167,3 +168,33 @@ def has_filters(query_params, exclude=["order", "per_page"]):
     be they query string, court, date, or party.
     """
     return len(set(k for (k, v) in query_params.items() if v) - set(exclude)) > 0
+
+
+def parameter_provided(params, parameter_name):
+    value = params.get(parameter_name)
+    if value and len(value) > 0:
+        return True
+
+
+def parse_parameter_as_int(params, parameter_name, default=None):
+    if parameter_provided(params, parameter_name):
+        return int(params.get(parameter_name))
+    else:
+        return default
+
+
+def parse_date_parameter(params, param_name, default_to_last=False):
+    year_param_name = f"{param_name}_year"
+    month_param_name = f"{param_name}_month"
+    day_param_name = f"{param_name}_day"
+    if parameter_provided(params, param_name):
+        return params[param_name]
+    elif parameter_provided(params, year_param_name):
+        year = parse_parameter_as_int(params, year_param_name, default=1)
+        default_month = 12 if default_to_last else 1
+        month = parse_parameter_as_int(params, month_param_name, default=default_month)
+        default_day = monthrange(year, month)[1] if default_to_last else 1
+
```

Comment:
for consistency with the year and month fields being next to each other, can we remove this newline between these 2 lines?